opentelemetry-collector/templates/configmap.yaml

```diff
@@ -22,6 +22,7 @@ data:
         api_url: {{ .Values.honeycomb.apiHost }}
         dataset: {{ .Values.honeycomb.dataset }}
         sample_rate: {{ .Values.honeycomb.sample_rate }}
+        sample_rate_attribute: {{ .Values.honeycomb.sample_rate_attribute }}
```

Comment:
Does this matter if the sample_rate_attribute by default is commented out?

Reply:
It doesn't matter, a blank will be output, and the exporter's default will handle it, but I did want to remove the sample_rate_attribute property from this PR, and clearly I missed an entry.